samples/qml/view.qml

```diff
@@ -6,6 +6,7 @@ import QtQuick 2.0
 import QtQuick.Controls 2.0
 import QtQuick.LocalStorage 2.0
 
+import "../../dist/glean.js" as GleanInstance;
```

Comment:
In line 7 we import the `LocalStorage` module, that is globally accessible from the `metrics.js` file. For some reason it is not globaly accessible from glean.js unless I import it in this file. It's kinda weird to import this here, but I guess it's ok for now.

Reply:
nvm, added a comment saying this.